examples/flow/flow_pois.py

```diff
@@ -58,6 +61,33 @@ def get_set(file_name):
 wrapper = LammpsWrapper()
 wrapper.CM[CUBA.NUMBEROF_TIME_STEPS] = 10000
 wrapper.CM[CUBA.TIME_STEP] = 0.003
+wrapper.CM[CUBA.PAIR_POTENTIALS] = ("lj:\n"
```

Comment:
I was also a little curious about the use of the `PAIR_POTENTIALS` since as @nathanfranklin commented this is the first implementation and might be ugly so I was wondering what was the plan. I am looking forward to see the next steps. 

In any case this was just a curiosity comment and thanks for taking the time to explain.